data/transport missions.txt

```diff
@@ -3404,3 +3404,69 @@ mission "Capture Smuggler"
 		payment 150000
 		conversation
 			`The huge man drags the unconscious smuggler out of the cargo container with one hand and carries the cargo box with the other. He stops in front of you. "<first> <last>," he says. "My employer, he will be very happy. Your payment, you will receive soon. Is it not good, to be reliable?" The man leaves your ship, letting the smuggler's legs scrape along the ground as he walks. A moment later, you receive <payment> from an anonymous account.`
+
+
+
+mission "Inheritance Redirection"
+	description "Travel to <destination> to act as heir for the late warlord Limping Pappa."
+	source
+		government "Pirate"
+	destination
+		distance 7 9
+		government "Republic" "Free Worlds" "Syndicate"
+	to offer
+		credits >= 400000
+	on accept
+		payment -200000
+	on offer
+		conversation
+			`You notice a computer terminal blinking in the corner of an alley.`
+			choice
+				`	(Check it out.)`
+				`	(Ignore it.)`
+					decline
+			`When you approach, you are greeted by a voice from the terminal; "Dear Sir, Madam, please respond!`
+			choice
+				`	"Hello"`
+				`	(Ignore it and walk away.)`
+					decline
+			`	"My name is Mr. Smith and I'm contacting you with a highly confidential and urgent proposal. I am, or more accurately was, the financial manager of the recently deceased warlord Limping Pappa. I am looking for a highly trustworthy individual who can help me relocate Mr. Pappa's fortune."`
+			`	"Mr. Pappa acquired over 300 million credits."`
```

Comment:
This seems to end pretty abruptly before entering the choice. Referring back to the above comment, I think this would be better absorbed into the previous paragraph and written as, `"I am looking for a highly trustworthy individual who can help me relocate Mr. Pappa's fortune of 300 million credits."`

Reply:
Fixed as suggested.